Modules/_ssl.c

```diff
@@ -4133,13 +4133,6 @@ _ssl__SSLContext_load_verify_locations_impl(PySSLContext *self,
             if (PyObject_GetBuffer(cadata, &buf, PyBUF_SIMPLE)) {
                 goto error;
             }
-            if (!PyBuffer_IsContiguous(&buf, 'C') || buf.ndim > 1) {
```

Comment:
Is the `buf.ndim > 1` check still important?

Reply:
You are right, I missed it.